### PR TITLE
Use id 12 to LongArrayTag, add unregistering to Registries

### PR DESCRIPTION
**Pull request by creeper123123321**
https://wiki.vg/NBT#Specification

### CODE REVIEW

**Comment by Steveice10** [comment]
Any particular reason behind adding the unregister methods?

**Comment by creeper123123321** [comment]
I want to disable the Serializable(Array)Tag because I heard that
deserializing objects from unknown sources may be insecure

File: src/main/java/com/github/steveice10/opennbt/conversion/ConverterRegistry.java [hunk 1]
```diff
@@ -99,6 +89,28 @@
...
+     *
+     * @param <T>  Tag type to unregister.
+     * @param <V>  Value type to unregister.
+     * @param tag  Tag type class to unregister.
+     * @param type Value type class to unregister.
+     * @throws ConverterUnregisterException If an error occurs while unregistering the converter.
+     */
+    public static <T extends Tag, V> void unregister(Class<T> tag, Class<V> type) throws ConverterUnregisterException {
+        if (!tagToConverter.containsKey(tag)) {
+            throw new TagUnregisterException("Type conversion to tag " + tag.getName() + " is not registered.");
```
Comment (Steveice10) [inline src/main/java/com/github/steveice10/opennbt/conversion/ConverterRegistry.java]: This (and the one below) should be ConverterUnregisterException

**Comment by Steveice10** [comment]
Fair enough. Might actually just remove the serializable tags instead then since they kind of defeat the purpose of a structured binary format. Also left a comment.

**Comment by Steveice10** [comment]
Pulled and made some additional changes:
* Ended up removing the new exceptions as it can just fail silently on removing something that was never registered.
* Removed the serializable tags for the reasons mentioned above, along with the string array tag since that can easily be replicated with a ListTag of StringTags.

**Pull request merged**